README.md

```diff
@@ -290,7 +290,8 @@ For that reason a comma between each member of a model is mandatory, unlike clas
 -   `.props(props)` produces a new type, based on the current one, and adds / overrides the specified properties
 -   `.actions(self => object literal with actions)` produces a new type, based on the current one, and adds / overrides the specified actions
 -   `.views(self => object literal with view functions)` produces a new type, based on the current one, and adds / overrides the specified view functions
--   `.preProcessSnapshot(snapshot => snapshot)` can be used to pre-process the raw JSON before instantiating a new model. See [Lifecycle hooks](#lifecycle-hooks-for-typesmodel)
+-   `.preProcessSnapshot(snapshot => snapshot)` (deprecated in favor of `types.snapshotProcessor`) can be used to pre-process the raw JSON before instantiating a new model. See [Lifecycle hooks](#lifecycle-hooks-for-typesmodel)
```

Comment:
If it simplifies code, I am pro bringing out a next major quickly. But it is nice to add in a minor first to see whether there are cases we can't handle

Reply:
sounds good, anyway this PR doesn't remove the old hooks (which I guess brings the eternal question, is something deprecated after it is no longer available or before? :) )

Reply:
renamed the wording so it is alternatively instead of deprecated